### ISSUE TITLE
Doesn't work anymore with v0.3.3

### ISSUE THREAD

**Issue by Legion2** [wdhongtw/vscode-gpg-indicator#7]
The icon in the status bar is missing and I can't unlock my key after the latest update.
But it worked before. I already reinstalled the extension but still does not show in the status bar.

**Comment by SitiBanc** [comment]
Same here. I checked the console and found this error message:
```
workbench.desktop.main.js:1857 Activating extension 'wdhongtw.gpg-indicator' failed: Cannot find module 'lookpath'
```

**Comment by wdhongtw** [comment]
Hi guys, sorry for the issue.

I may not able to solve this issue immediately, so the quick solution is to download 0.3.2 from the GitHub release page.
<https://github.com/wdhongtw/vscode-gpg-indicator/releases/tag/v0.3.2>
And in the Extension Tab in VS Code UI, select install from VSIX.

I'm investigating this now, sorry for the inconvenience. 😥

**Comment by wdhongtw** [comment]
Fixed in #8
